src/field_theory/intermediate_field.lean

```diff
@@ -274,6 +274,8 @@ def map (f : L →ₐ[K] L') : intermediate_field K L' :=
   neg_mem' := λ x hx, (S.to_subalgebra.map f).neg_mem hx,
   .. S.to_subalgebra.map f}
 
+@[simp] lemma coe_map (f : L →ₐ[K] L') : ↥(S.map f) = ↥(f '' S) := rfl
```

Comment:
If nothing else, I'd expect `coe_map` to be the lemma talking about `set` coercion, which also seems to be missing.